src/Impostor.Api/Innersloth/VentLocation.cs

```diff
@@ -0,0 +1,48 @@
+namespace Impostor.Api.Innersloth
+{
+    public enum VentLocation : uint
```

Comment:
Should these be separate enums? How does this work with them all being in the same enum?

Reply:
I think this is fine. If you write plugin code you can easily do 

```csharp
if (location == VentLocation.SkeldMedbay ||
    location == VentLocation.MiraMedbay ||
    location == VentLocation.PolusLaboratory) {

}
```